src/index.js

```diff
@@ -134,8 +138,12 @@ class Dropzone extends React.Component {
   }
 
   open() {
+    const clickEvent = new MouseEvent('click', {
+      bubbles: false
+    });
+
     this.fileInputEl.value = null;
-    this.fileInputEl.click();
+    this.fileInputEl.dispatchEvent(clickEvent);
```

Comment:
Why not just cancel bubbling in the event handler then? This looks like a hack to me.